fastlane/Fastfile

```diff
@@ -229,9 +219,42 @@ platform :android do
     end
   end
 
+  desc "Build and run purchases module custom entitlement computation integration tests"
+  desc "This requires the google cloud cli to be installed and initialized."
+  lane :run_custom_entitlement_computation_integration_tests do |options|
+    app_name = 'customEntitlementComputationIntegrationTest'
+    build_purchases_integration_tests(
+      app_name: app_name,
+      api_key: ENV['CUSTOM_ENTITLEMENT_COMPUTATION_REVENUECAT_API_KEY'],
+      google_purchase_token: ENV['CUSTOM_ENTITLEMENT_COMPUTATION_GOOGLE_PURCHASE_TOKEN'],
+      product_id_to_purchase: ENV['CUSTOM_ENTITLEMENT_COMPUTATION_PRODUCT_ID_TO_PURCHASE'],
+      base_plan_id_to_purchase: ENV['CUSTOM_ENTITLEMENT_COMPUTATION_BASE_PLAN_ID_TO_PURCHASE'],
+      active_entitlements_to_verify: ENV['CUSTOM_ENTITLEMENT_COMPUTATION_ACTIVE_ENTITLEMENTS_TO_VERIFY'] || '',
```

Comment:
why have an empty default for this one?

Reply:
That's the value we currently use... Since we use an expired token, there are no active entitlements when checking. Need to have an active token in order to get active entitlements we can check here. 

Currently, this is only useful for the load shedder integration tests which do grant entitlements since we don't check against Google

Reply:
feels like something that's worth adding a comment for - a maintainer could very reasonable remove this and assume it's okay